common/src/main/resources/schema/datasources_1_4.xsd

```diff
@@ -724,6 +724,17 @@
           </xs:documentation>
         </xs:annotation>
       </xs:element>
+      <xs:element name="fair" type="xs:boolean" minOccurs="0">
```

Comment:
jesper, please can you clarify what you mean by this comment.  Should the order of the elements in the toString() methods in the class above (DsXaPoolImpl, DsPoolImpl & XaPoolImpl) be the same as the order of elements in datasources_1_4.xsd?  At present they are, but they are before use-strict-min.  This comments suggests that it is ok to be before if the fair elements in the xsd are before use-strict-min elements, which is how they are currently. The previous comments suggest that fair needs to be after use-strict-min